shared/js/media/remote_controls.js

```diff
@@ -197,11 +180,6 @@ MediaRemoteControls.prototype._setupBluetooth = function(callback) {
       self._commandHandler(AVRCP.PLAY_PRESS);
     }
   }
-
-  function resetDefaultAdapter() {
-    self.defaultAdapter = null;
-    // Do we need to do anything else?
-  }
```

Comment:
Though it might not be necessary, but does the bluetooth helper reset the adapter for us if we have errors on getting it?